dead_mans_switch_client/models/dead_mans_switch_client.py

```diff
@@ -53,6 +55,10 @@ def alive(self):
         if not url:
             logger.error('No server configured!')
             return
+        timeout = self.env['ir.config_parameter'].get_param(
+            'dead_mans_switch_client.send_timeout')
```

Comment:
just put the default as second parameter, then you can delete the following two lines